src/main/java/mate/academy/spring/model/Skill.java

```diff
@@ -0,0 +1,66 @@
+package mate.academy.spring.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+import java.util.Set;
```

Comment:
javax and java are differrent imports, so they must  separate by empty line

Reply:
@DmitriyLitvin  Ok